lib/chewy/search/loader.rb

```diff
@@ -53,7 +53,7 @@ def load(hits)
           type = derive_type(index_name, type_name)
           ids = hit_group.map { |hit| hit['_id'] }
           loaded = type.adapter.load(ids, @options.merge(_type: type))
-          loaded ||= hit_group.map { |hit| type.build(hit) }
+          loaded = type.adapter.load(ids, **@options.merge(_type: type))
```

Comment:
i don't understand this change. i think you will override loaded

Reply:
Thanks This was an Oversight
It has been fixed and PR updated